mw-webapp/src/component/select/Select.tsx

```diff
@@ -0,0 +1,74 @@
+import {Option} from "./option/Option";
+import styles from "src/component/select/Select.module.scss";
+
+/**
+ * Option element for a select component
+ */
+export interface OptionType {
```

Comment:
Why interface for options inside select component? Check please

Reply:
done